src/test/ui/generic/generic-no-mangle.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(improper_ctypes)]
```

Comment:
Curious why this is a top level attribute, I only see one function I'd expect to cause a warning (`bar`).